examples/python/tools/exa-tool/chile_government_services_assistant_.py

```
@@ -0,0 +1,141 @@
+# {page_url}
+
+# ## Content:
+# {page_content}
+
+# """
+
+# """# Initialize Firecrawl Tool"""
+
+firecrawl_tool = FirecrawlTool(
+    api_key=os.environ['FIRECRAWL_API_KEY'],
+    instruction=FIRECRAWL_INSTRUCTION,
```

Ruff (0.11.9)

Undefined name `FIRECRAWL_INSTRUCTION`

(F821)